include/unifex/this.hpp

```diff
@@ -16,8 +16,7 @@
 #pragma once
 
 #include <unifex/config.hpp>
-
-#include <type_traits>
```

Comment:
Similarly here, still using `std::` traits below.
Should this #include remain? Or are you assuming that `unifex/type_traits.hpp` will transitively include it?

Reply:
That.